spring-boot/starter/src/main/java/org/togglz/spring/boot/actuate/TogglzEndpoint.java

```diff
@@ -36,9 +39,14 @@
 @Endpoint(id = "togglz")
 public class TogglzEndpoint  {
 
+    private final FeatureProvider featureProvider;
+
```

Comment:
I don't think we need `FeatureProvider`. You can get the list of features with `featureManager.getFeatures()`, like in `getAllFeatures` methods.